srcpkgs/Aegisub/template

```diff
@@ -1,61 +1,44 @@
 # Template file for 'Aegisub'
 pkgname=Aegisub
-version=3.2.2
-revision=12
-build_style=gnu-configure
-configure_args="--disable-update-checker --with-alsa --with-ffms2
- --with-wx-config=wx-config-gtk3 $(vopt_with fftw fftw3) $(vopt_with openal)
- $(vopt_with portaudio)"
-hostmakedepends="automake gettext-devel intltool libtool LuaJIT pkg-config"
+version=3.3.2
+revision=1
+_githash=723d01d13088d9b1bccc821f1129d045a0b74e2c
+build_style=cmake
+cmake_builddir="BUILD"
+configure_args="
+	-DwxWidgets_CONFIG_EXECUTABLE=/usr/bin/wx-config-gtk3
+	-DWITH_PORTAUDIO=$(vopt_if portaudio ON OFF)
+	-DWITH_OPENAL=$(vopt_if openal ON OFF)
+	-DWITH_FFTW3=$(vopt_if fftw ON OFF)
+"
+hostmakedepends="automake gettext-devel intltool libtool LuaJIT pkg-config git"
 makedepends="alsa-lib-devel boost-devel libcurl-devel fontconfig-devel
- freetype-devel hunspell-devel icu-devel libass-devel libffms2-devel LuaJIT-devel
- MesaLib-devel wxWidgets-gtk3-devel $(vopt_if fftw fftw-devel)
- $(vopt_if openal libopenal-devel) $(vopt_if portaudio portaudio-devel)"
-short_desc="Tool for creating and modifying subtitles"
-maintainer="Enno Boland <gottox@voidlinux.org>"
+	freetype-devel hunspell-devel icu-devel libass-devel libffms2-devel
+	LuaJIT-devel MesaLib-devel wxWidgets-gtk3-devel $(vopt_if fftw
+	fftw-devel) $(vopt_if openal libopenal-devel) $(vopt_if portaudio
+	portaudio-devel)"
```

Comment:
Multiline strings should be indented with spaces, and don't leave the terminating quote in a new line.

Reply:
done!

Reply:
okay looks like the linter wants me to indent with one space, not two...